src/components/BookSearch/SearchBar.js

```diff
@@ -0,0 +1,53 @@
+import Button from 'react-bootstrap/Button';
+import Container from 'react-bootstrap/Container';
+import Form from 'react-bootstrap/Form';
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+import { useState } from 'react';
+import "../styles/search-bar.css";
+
+function SearchBar({ onSubmit }) {
```

Comment:
Ten komponent to Nav bar Zmieńmy tą nazwe tutaj dla czytelności

Reply:
Fakt